server/views/pages/overview/index.njk

```diff
@@ -19,6 +19,17 @@
 
 {% block content %}
 
+  {% if tab === 'overview' and showGoalCreationSuccessMessage %}
+    {{ mojBanner({
```

Comment:
Personally (IMHO) I think the banner in the design looks a little ... errr ... meh! 🤷‍♂️ 
I think I'd rather see the regular GDS Notification banner, but what do I know? 🤣 

Our design team have assured me [this banner style is part of the MoJ Design System](https://design-patterns.service.justice.gov.uk/components/banner#success), so I guess its OK 🤷‍♂️